sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java

```diff
@@ -1215,4 +1232,25 @@ public Boolean isSampleEditAllowed(String sampleUuid) {
 
 		return sampleService.isSampleEditAllowed(sample);
 	}
+
+	@Override
+	public Disease getDiseaseOf(SampleDto sample) {
```

Comment:
Could be moved to the UI module, don't really see a reason sending SampleDto back to backend